### ISSUE TITLE
InvalidOperationException: Id must be used with Drop Down Manager

### ISSUE THREAD

**Issue by endeffects** [chanan/BlazorStrap#89]
I added your navigation control today, but i'm receiving an error when i click more than one link:

> InvalidOperationException: Id must be used with Drop Down Manager

```
    <BlazorNav IsVertical="true">
        <NavItem>
            <BlazorNavLink href="#" Match="NavLinkMatch.All">Home</BlazorNavLink>
        </NavItem>
        <strong>CategoryOne</strong>
        <NavItem>
            <BlazorNavLink href="CategoryOne/Create">Create</BlazorNavLink>
        </NavItem>
        <NavItem>
            <BlazorNavLink href="CategoryOne/Delete">Delete</BlazorNavLink>
        </NavItem>
        <NavItem>
            <BlazorNavLink href="CategoryOne/Rename">Rename</BlazorNavLink>
        </NavItem>
        <NavItem>
            <BlazorNavLink href="CategoryOne/Show">Show</BlazorNavLink>
        </NavItem>
        <strong>CategoryTwo</strong>
        <NavItem>
            <BlazorNavLink href="CategoryTwo/Create">Create</BlazorNavLink>
        </NavItem>
        <NavItem>
            <BlazorNavLink href="CategoryTwo/Delete">Delete</BlazorNavLink>
        </NavItem>
        <NavItem>
            <BlazorNavLink href="CategoryTwo/Rename">Rename</BlazorNavLink>
        </NavItem>
        <NavItem>
            <BlazorNavLink href="CategoryTwo/Move">Move</BlazorNavLink>
        </NavItem>
        <NavItem>
            <BlazorNavLink href="CategoryTwo/Show">Show</BlazorNavLink>
        </NavItem>
    </BlazorNav>
```

![image](https://user-images.githubusercontent.com/13432569/60768498-143c2c00-a0c5-11e9-946f-bd55f069a4b4.png)

Also i'd like to know what is the best practice to have a category based navigation without dropdown menus.

**Comment by jbomhold3** [comment]
Confirmed and addressing

**Comment by jbomhold3** [comment]
Fixed with https://github.com/chanan/BlazorStrap/pull/91

**Comment by chanan** [comment]
Fixed in 1.0.0-preview6-18
